src/Cart-components/CartSummary.tsx

```diff
@@ -0,0 +1,43 @@
+import { useCartContext } from '../Context/Cart';
+
+const CartSummary: React.FC = () => {
+    const { state: { totalPrice } } = useCartContext(); 
+    const remainingForFreeShipping = 35 - totalPrice;
+
+    const deliveryCharge = totalPrice < 35 ? 5 : 0; 
```

Comment:
This is a good idea! Ideally we would fetch the delivery prices from the database (it sin the table called shipping_options). It will give you all the options available to list as well as lead time in days
![image](https://github.com/user-attachments/assets/78f367f3-6afa-4ed9-8b54-a4866df9c846)